src/schema/artist/related.ts

```diff
@@ -43,6 +48,9 @@ export const Related = {
             type: GraphQLBoolean,
             defaultValue: true,
           },
+          min_forsale_artworks: {
```

Comment:
The other param to this resolver is `exclude_artists_without_artworks`, which isn't camel case and instead just matches the underlying gravity param.

Today, I don't know that we would necessarily abide by that (we prefer camel case, we don't need to map params directly, we can have our own client schema, etc.), but thought mixing styles in the same existing resolver would be odd, so kept it old-school here.